.github/workflows/docs-skip.yaml

```diff
@@ -0,0 +1,91 @@
+name: Docs skip checks
+
+# Workflow to skip steps required for branch protection but not required for documentation changes.
+#
+# Add the job there with 'skip-' prefix and the step will just echo it is not required. 
+# The 'name' of the step MUST match the original step we like to skip
+#
+# Example:
+#
+# For the following step on a workflow that ignore docs but required for branch protection - 
+#
+# test-something:
+#    name: Testing something
+#    steps:
+#      ...
+#
+# We will add a skip step here -
+#
+# skip-test-something:
+#    name: Testing something
+#    steps:
+#      - run: 'echo "Not required"'
+#
+
+on:
```

Comment:
I think I understand this workflow but I didn't quite get why do we need the paths-ignore here?

Reply:
@N-o-Z thanks, my bad the above will not work as I don't have a way to have include only on path changes or when other workflow is skipped. Thanks for your time. I'll close this one for now.